docs/source/algorithms/ConvertPeaksWorkspace-v1.rst

```diff
@@ -0,0 +1,56 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm provides bi-directional conversion bewteen regular PeaksWorkspace and LeanElasticPeaksWorkspace.
+
+When casting a PeaksWorkspace into a LeanElasticPeaksWorkspace, information related to detector
+(DetID, BankName, Row, Col) is discarded.
+However, other experiment information are retained in the LeanElasticPeaksWorkspace, including
+run, hkl, goniometer, sample, as well as the corresponding information at the invidual Peak level.
```

Comment:
"invidual" -> "individual"

Reply:
the typo will be fixed in the next commit